policies/templates/gcp_gke_private_cluster_v1.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check to see if the GKE Cluster is a private cluster.
+# Part of https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#restrict_cluster_discovery_rbac_permissions
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-private-cluster-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPGKEPrivateClusterConstraintV1
+        plural: gcpgkeprivateclusterconstraintsv1        
+      validation:
+        openAPIV3Schema:
+          properties: {}
+  targets:
+    validation.gcp.forsetisecurity.org:
```

Comment:
let's update the target format

Reply:
What should i update this to?

Reply:
discussed over chat. we'll bulk update the targets in a separate PR